src/casp/services/_settings.py

```diff
@@ -0,0 +1,73 @@
+import json
+import os
+import typing as t
+
+import dotenv
+from pydantic import BaseSettings
+
+dotenv.load_dotenv(dotenv_path="casp/services/.env")
+
+ENV_TYPE = os.environ.get("ENVIRONMENT")
+
+
+class AllEnvSettings(BaseSettings):
+    # General
+    GOOGLE_ACCOUNT_CREDENTIALS: t.Dict = json.loads(os.environ.get("GOOGLE_SERVICE_ACCOUNT_CREDENTIALS", "{}"))
+    ENVIRONMENT: str = os.environ.get("ENVIRONMENT")
+    # PCA Model serving
+    PCA_MODEL_BUCKET_NAME: str = "fedor-test-bucket"
+    PCA_MODEL_BLOB_NAME: str = "models/dump_manager_002.pickle"
+    # API
+    SERVER_HOST: str = "0.0.0.0"
+    SERVER_PORT: int = 8000
+    MODEL_SERVER_URL: str = "https://casp-pca-serving-vi7nxpvk7a-uc.a.run.app/predict"
```

Comment:
Could the default values for all of these be read from a JSON file? It feels bit sketchy to have e.g. pointers to specific Cloud Run deployments in the codebase...